go/consensus/tendermint/apps/registry/transactions.go

```diff
@@ -185,6 +181,20 @@ func (app *registryApplication) registerNode( // nolint: gocyclo
 		return err
 	}
 
+	if ctx.IsCheckOnly() {
+		// This method is exempt from the minimum balance requirement. We'll
+		// at least check that an entity allows this node though.
+		if !untrustedEntity.HasNode(untrustedNode.ID) {
```

Comment:
I added this check, but an attacker could still repeatedly update their entity to allow new node accounts and then using RegisterNode to create new node accounts with nonces

Reply:
we're removing this too